application-github-statistics-ui/src/main/resources/GitHubStats/CommittersMacro.xml

```diff
@@ -20,75 +20,228 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc>
+<xwikidoc version="1.3" reference="GitHubStats.CommittersMacro" locale="">
```

Comment:
hmm maybe not.... In which case, we could increase the minimum required version to 8.4 for the new version. I'd be fine with that. Especially since it's not a highly used app to say the least ;)

Still it's a good idea to make modifications on the minimum version and to export from it.